src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
 // write code here
+document.addEventListener('click', (e) => {
+  const clickedElement = e.target;
+  const list = clickedElement.querySelector('ul');
```

Comment:
The method `querySelector` is used here, which will only find the first matching `ul` element within the `clickedElement`. If the intention is to toggle the display of a specific `ul` element, ensure that the correct element is being targeted. If multiple `ul` elements need to be toggled, consider using `querySelectorAll` and iterating over the results.